### ISSUE TITLE
How did you normalize the value of loss to 1

### ISSUE THREAD

**Issue by Ir1d** [aiff22/PyNET-PyTorch#1]
![image](https://user-images.githubusercontent.com/10709657/75446963-63600580-59a3-11ea-94de-05f8009e7774.png)

Hi, thank you for your nice work. I had a small question reading your code. Please guide me on this. You mentioned that the value is normalized to 1, but I couldn't find the corresponding code. I'm curious about how this is implemented, because I haven't met this before. 
Thx in advance.

**Comment by Jam-G** [comment]
Hi Ir1d,
Here is my understanding， hopes it may helpful for you：
The papers said that
>>It is trained using the following loss function:
>> L_Level 1 = L_VGG+0.75*L_SSIM+0.05*L_MSE;
>>where the value of each loss is normalized to 1

It means L_vgg, L_ssim, L_mse are normalized to 1   respectively， but not total loss L_level1 is normalized to 1.  And the torch.nn.MSELoss() is normalized by default, the SSIM/MSSSIM implements are normalized to 1 by defination.

**Comment by Ir1d** [comment]
Thank you @Jam-G , I understand that they should be normalized respectively. But why is `MSELoss()` normalized by defualt?
What does `normalized to 1` mean ?

**Comment by aiff22** [comment]
Hi @Ir1d,

In the paper, ``normalized to 1`` is used to show the actual contribution of the loss functions to the total loss (which is around 55.5%, 42.5% and 3% for L_vgg, L_ssim and L_mse losses, respectively).

In other words, if the value of the total loss is equal to 1, then ~0.55 should correspond to the VGG loss, ~0.42 to the SSIM loss and ~0.03 to the MSE. This is done to make the loss function independent of any particular normalization used for each sub-loss.

**Comment by Ir1d** [comment]
@aiff22 Thank you for your clarification. Could you guide me the code to implement this normalization? I didn't notice related code in the repo.

**Comment by aiff22** [comment]
@Ir1d,

The weight coefficients of the loss functions were already chosen to approximately maintain the above contribution of the losses, so no further code changes are needed (please note that in PyTorch implementation a slightly different combination of the loss functions is used).

**Comment by Ir1d** [comment]
Thank you @aiff22
